contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -356,27 +365,22 @@ abstract contract PoSValidatorManager is
 
         Validator memory validator = getValidator(validationID);
 
-        // The received nonce should be no greater than the highest sent nonce. This should never
-        // happen since the staking manager is the only entity that can trigger a weight update
-        // on the P-Chain.
-        require(validator.messageNonce >= nonce, "PoSValidatorManager: invalid nonce");
-
-        // The nonce should also be greater than or equal to the delegationID's starting nonce. This allows
-        // a weight update using a higher nonce (which implicitly includes the delegation's weight update)
-        // to be used to complete registration for an earlier delegation. This is necessary because the P-Chain
-        // is only willing to sign the latest weight update.
-        require(
-            $._delegatorStakes[delegationID].startingNonce <= nonce,
-            "PoSValidatorManager: nonce does not match"
-        );
+        // The received nonce should be no greater than the highest sent nonce, and at least as high as
+        // the delegation's statubg nonce. This allows //a weight update using a higher nonce
+        // (which implicitly includes the delegation's weight update) to be used to complete delisting
+        // for an earlier delegation. This is necessary because the P-Chainis only willing to sign the latest weight update.
```

Comment:
```suggestion
        // the delegation's starting nonce. This allows a weight update using a higher nonce
        // (which implicitly includes the delegation's weight update) to be used to complete delisting
        // for an earlier delegation. This is necessary because the P-Chain is only willing to sign the latest weight update.
```